addon/mixins/hot-component.js

```diff
@@ -5,10 +5,12 @@ export default Ember.Mixin.create({
   init () {
```

Comment:
I like it - not a fan of using the mixin these days unless is literally the only option so I'm glad to see this go away